web/pom.xml

```diff
@@ -100,6 +100,18 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.jaxrs</groupId>
```

Comment:
I take that not having these dependencies where the only thing preventing the json based requests from working, right?